arviz/rcparams.py

```diff
@@ -93,11 +102,39 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
+def make_iterable_validator(scalar_validator, length=None, allow_none=False, allow_auto=False):
+    """Validate value is an iterable datatype."""
+    # based on matplotlib's _listify_validator function
+
+    def validate_iterable(value):
+        if allow_none and (value is None or isinstance(value, str) and value.lower() == "none"):
+            return None
+        if isinstance(value, str):
+            if allow_auto and value.lower() == "auto":
+                return "auto"
+            value = tuple(v.strip("([ ])") for v in value.split(",") if v.strip())
+        if np.iterable(value) and not isinstance(value, (set, frozenset)):
```

Comment:
`not isinstance(value, (str, set, frozenset)):` see other comment. In matplotlib's function there was an elif here, so strings did not get to this point. I think adding this here will keep the desired behaviour.

Reply:
Does this function need to work with a string value like ```"(1, 4)"```? If not, then for string we can only keep the check for ```"auto"``` and remove line 115 that makes the tuple right? While also adding the str check along with set and frozen set.

Reply:
Yes, it should work with strings like `"1,4"`, `"(1, 4)"` or with square brackets. This strings are what would be read if the user set a tuple in the rcparams file